main.go

```diff
@@ -100,23 +103,25 @@ func main() {
 	setupLog := log.Log
 
 	var (
-		configFile string
-		err        error
+		metricsAddr          string
+		enableLeaderElection bool
+		probeAddr            string
+		err                  error
 	)
-	flag.StringVar(&configFile, "config", "",
-		"The operator will load its initial configuration from this file. "+
-			"Omit this flag to use the default configuration values. "+
-			"Command-line flags override configuration from this file.")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
```

Comment:
I wonder if the deployment of the operator should change after this change. Or defaults are good enough?

Reply:
I think these defaults are good. These are the same as https://github.com/Kuadrant/kuadrant-operator/commit/52b2cdb62e3594f39b353665428cf5531cce574c which was before it was moved to loading config using file.

Only difference I see between defaults and deployment options is that `leacterElection` is `true` only for deployment but `false` by default 🤔 :
- https://github.com/Kuadrant/kuadrant-operator/pull/246/files#diff-ef2df99e0c13ecb3c10ae6aa1923d7892fdccfa82e8f8777faebdaa4528c7568L1-L11
- https://github.com/Kuadrant/kuadrant-operator/pull/246/files#diff-319f8dc9adf6d477651de0cb44f85c968909535fc7c3a61be091a08b68298732R12

Reply:
But the deployment is setting it explicitly, so we should be good.